### ISSUE TITLE
NameError: name 'numpy' is not defined

### ISSUE THREAD

**Issue by igor-yusupov** [rosefun/SemiSupervised#3]
TSVM.py, line 140, in predict
you need use np.array instead of numpy.array

**Comment by rosefun** [comment]
Yes, this problem exists in version `0.0.26` and `0.0.27`. Please install the latest version `pip install semisupervised==0.0.28`.

**Comment by ranjiGT** [comment]
How can I modify this code for a multi-class problem for image classification?
